udata/core/spatial/models.py

```diff
@@ -64,6 +87,17 @@ def keys_values(self):
                 keys_values.append(value)
         return keys_values
 
+    @property
+    def url(self):
+        return url_for('gouvfr.territory', territory=self)
```

Comment:
That's where it's starting to be tricky. These properties are only called when the gouvfr extension is activated…

I'm open to better suggestions :/